src/components/common/Extensions/UserHub/partials/TransactionsTab/TransactionsTab.css

```diff
@@ -1,35 +0,0 @@
-@value query700 from "~styles/queries.css";
```

Comment:
Is deleting the entire file intentional? There are other styles here for the scrollbar (vertical), which might be needed.

Reply:
In fact, scrollbar styles might still apply from `main.global.css`.